<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMWidget</class>
 <widget class="QWidget" name="AMWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QwtPlot" name="plot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBW">
     <property name="title">
      <string>Filter Bandwidth</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="textLabelBandWidth">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabelBandWidth</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderBandwidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pageStep">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_10">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxDemodulation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Demodulation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QRadioButton" name="radioButtonDemAM">
          <property name="text">
           <string>AM</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDemodulation</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDemLSB">
          <property name="text">
           <string>LSB</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDemodulation</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDemUSB">
          <property name="text">
           <string>USB</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDemodulation</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDemCW">
          <property name="text">
           <string>CW</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDemodulation</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDemFM">
          <property name="text">
           <string>FM</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupDemodulation</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxAGC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>AGC</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QRadioButton" name="radioButtonAGCOff">
          <property name="text">
           <string>Off</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAGC</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonAGCSlow">
          <property name="text">
           <string>Slow</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAGC</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonAGCMed">
          <property name="text">
           <string>Med</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAGC</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonAGCFast">
          <property name="text">
           <string>Fast</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupAGC</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxNoiseReduction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Noise Reduction</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonNoiRedOff">
          <property name="text">
           <string>Off</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupNoiseReduction</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonNoiRedLow">
          <property name="text">
           <string>Low</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupNoiseReduction</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButtonNoiRedMed">
          <property name="text">
           <string>Med</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupNoiseReduction</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="radioButtonNoiRedHigh">
          <property name="text">
           <string>High</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupNoiseReduction</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonNoiRedSpeex">
          <property name="text">
           <string>Speex</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupNoiseReduction</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxNoiRedLevel">
          <property name="suffix">
           <string> dB</string>
          </property>
          <property name="minimum">
           <number>-96</number>
          </property>
          <property name="maximum">
           <number>-1</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QGroupBox" name="GroupBoxPLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>PLL</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxPLL">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GroupBoxAutoFreqAcq">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Auto Frequency Acquisition</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxAutoFreqAcq">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPlot">
       <property name="title">
        <string>Plot</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxWaterFall">
          <property name="text">
           <string>Waterfall</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Carrier Offset</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelFreqOffset">
          <property name="text">
           <string>Carrier Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonFreqOffset">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="Spacer6_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupNoiseReduction"/>
  <buttongroup name="buttonGroupAGC"/>
  <buttongroup name="buttonGroupDemodulation"/>
 </buttongroups>
</ui>
